action.yml

```diff
@@ -60,11 +60,23 @@ runs:
           fi
         elif [ "${{ runner.os }}" = "macOS" ]; then
           brew install ninja ccache qemu dtc
-        elif [ "${{ runner.os }}" = "Windows" ]; then
-          choco feature enable -n allowGlobalConfirmation
-          choco install ninja wget
         fi
 
+    - name: Install Windows dependencies
+      if: ${{ runner.os == "Windows" }}
+      shell: pwsh
+      # https://stackoverflow.com/a/75794127/337201
+      run: |
+          $ProgressPreference = 'SilentlyContinue'
+          Add-AppxPackage 'https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx'
+          Invoke-WebRequest -Uri https://github.com/microsoft/winget-cli/releases/latest/download/DesktopAppInstaller_Dependencies.zip -OutFile .\DesktopAppInstaller_Dependencies.zip
+          Expand-Archive .\DesktopAppInstaller_Dependencies.zip
+          Add-AppxPackage .\DesktopAppInstaller_Dependencies\x64\Microsoft.VCLibs.140.00.UWPDesktop_14.0.33728.0_x64.appx
+          Add-AppxPackage .\DesktopAppInstaller_Dependencies\x64\Microsoft.UI.Xaml.2.8_8.2310.30001.0_x64.appx
+          Invoke-WebRequest -Uri https://github.com/microsoft/winget-cli/releases/latest/download/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle -OutFile winget.msixbundle
+          Add-AppxPackage .\winget.msixbundle
+          nuget install Ninja-build.Ninja oss-winget.dtc wget
```

Comment:
Is it still `nuget` after installing `winget`?

Reply:
no, this was a silly test :)